src/test/java/pages/administrator/AcceptedTabPage.java

```diff
@@ -0,0 +1,43 @@
+package pages.administrator;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import pages.administrator.order.OrderComponent;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+
+public class AcceptedTabPage {
+
+    private final WebDriver driver;
+    private List<OrderComponent> orders;
+
+    public AcceptedTabPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(this.driver, this);
+        initOrderList();
```

Comment:
please move `initOrderList();` closer to its usage

Reply:
Done